mkdocs.yml

```diff
@@ -28,6 +28,7 @@ nav:
     - api-reference/dtypes.md
     - api-reference/dependencies.md
     - api-reference/selectors.md
+    - api-reference/group_by.md
```

Comment:
I wonder if we should order these alphabetically.. ?

Reply:
thanks for the comment. I sorted alphabetically `API Reference` and submitted PR #245